content/guides/guides/network-requests.md

```diff
@@ -273,13 +273,23 @@ Cypress you might want to check that out first.
 
 Here is an example of aliasing requests and then subsequently waiting on them:
 
-```javascript
+:::visit-mount-test-example
+
+```js
+cy.visit('http://localhost:8888/dashboard')
+```
+
+```js
+cy.mount(<Dashboard />)
+```
+
+```js
 cy.intercept('/activities/*', { fixture: 'activities' }).as('getActivities')
 cy.intercept('/messages/*', { fixture: 'messages' }).as('getMessages')
 
-// visit the dashboard, which should make requests that match
+// visiting or mounting the dashboard should make requests that match
```

Comment:
Can you make a separate comment for Visiting and Mounting and move it into the visit- and mount- specific code blocks above?